### PR TITLE
multi bare metal

### PR DESCRIPTION
**Pull request by veezhang**
This pr is base on #24 
Feature: #23 

- [x]  add cluster metrics for bare metal
- [x]  config file optimization
- [x]  grafana config
- [x]  helm charts adjustment
- [x] docs[en, cn] Fix: #19

### CODE REVIEW

File: deploy/kubernetes/README-CN.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+| `ingress.paths` | Ingress 的 paths | `["/metrics"]` |
+| `ingress.tls` | Ingress 的 tls | `[]` |
+| `ingress.ingressClass` | Ingress 的 ingressClass | `""` |
+| `podAnnotations` | Nebula stats exporter pod annotations | `{}` |
+| `podLabels` | Nebula stats exporter pod labels | `{}` |
+| `podLabels` | Nebula stats exporter resources | `{}` |
+| `livenessProbe` | Nebula stats exporter livenessProbe | `{"failureThreshold":2,"httpGet":{"path":"/health","scheme":"HTTP"},"initialDelaySeconds":30,"timeoutSeconds":10}` |
+| `nodeSelector` | Nebula stats exporter nodeSelector | `{}` |
+| `tolerations` | Nebula stats exporter tolerations | `{}` |
+| `affinity` | Nebula stats exporter affinity | `{}` |
```
Comment (MegaByte875) [inline deploy/kubernetes/README-CN.md:107]: Add more description relate k8s to make it clearly?

**Review by MegaByte875 — DISMISSED** [review]

File: deploy/kubernetes/example-CN.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+## 导入查看 grafana
+
+转发 grafana 服务：
+
+```shell
+$ kubectl -n monitoring port-forward svc/prometheus-grafana 3000:80
+```
+
+Grafana 的默认 `用户名/密码` 是 `admin/prom-operator`, http 地址是 `http://127.0.0.1:3000`.
```
[the remark this answers is not in the document]
Comment (veezhang) [reply deploy/kubernetes/example-CN.md:80]: The password for [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack) is `prom-operator`.

File: deploy/kubernetes/example-CN.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+$ kubectl delete crd alertmanagerconfigs.monitoring.coreos.com
+$ kubectl delete crd alertmanagers.monitoring.coreos.com
+$ kubectl delete crd podmonitors.monitoring.coreos.com
+$ kubectl delete crd probes.monitoring.coreos.com
+$ kubectl delete crd prometheuses.monitoring.coreos.com
+$ kubectl delete crd prometheusrules.monitoring.coreos.com
+$ kubectl delete crd servicemonitors.monitoring.coreos.com
+$ kubectl delete crd thanosrulers.monitoring.coreos.com
+$ kubectl delete ns monitoring example1 example1
+```
```
[the remark this answers is not in the document]
Comment (veezhang) [reply deploy/kubernetes/example-CN.md:107]: Shell script file ? Or shell command in markdown?
If necessary, merge later.

File: README-CN.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+## 基本的 Prometheus 配置
+
+在 prometheus.yaml 配置文件的 `scrape_configs` 中添加一个配置块：
+
+```yaml
+scrape_configs:
+  - job_name: 'nebula-stats-exporter'
+    static_configs:
+      - targets: ['<<NEBULA_STATS_EXPORTER_HOSTNAME>>:9100'] # nebula-stats-exporter metrics endpoints
```
[the remark this answers is not in the document]
Comment (veezhang) [reply README-CN.md:69]: If add a placeholder to the port, will it be too complicated?

File: deploy/kubernetes/README-CN.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+| `serviceMonitor.prometheusServiceMatchLabels` | Prometheus 的 ServiceMatchLabels, 使用 `kubectl -n `${namespace}` get prometheus `${name}` -o=jsonpath='{.spec.serviceMonitorSelector.matchLabels}'` 获取。| `{}` |
+| `ingress.enabled` | 是否开启 ingress | `false` |
+| `ingress.annotations` | Ingress 的 annotations | `{}` |
+| `ingress.hosts` | Ingress 的 hosts | `chart-example.local` |
+| `ingress.paths` | Ingress 的 paths | `["/metrics"]` |
+| `ingress.tls` | Ingress 的 tls | `[]` |
+| `ingress.ingressClass` | Ingress 的 ingressClass | `""` |
+| `podAnnotations` | Nebula stats exporter pod annotations | `{}` |
+| `podLabels` | Nebula stats exporter pod labels | `{}` |
+| `podLabels` | Nebula stats exporter resources | `{}` |
```
[the remark this answers is not in the document]
Comment (veezhang) [reply deploy/kubernetes/README-CN.md:103]: I've removed

File: deploy/kubernetes/README-CN.md (hunk 1 above)
Comment (veezhang) [reply deploy/kubernetes/README-CN.md:107]: Those who use operator should know the probabilities related to kubernetes. Do we need to reiterate it?

**Review by MegaByte875 — APPROVED** [review]
LGTM

**Pull request merged**